src/aiohomeconnect/cli/__init__.py

```diff
@@ -104,8 +112,17 @@ def subscribe_all_appliances_events(client_id: str, client_secret: str) -> None:
 async def _subscribe_all_appliances_events(client_id: str, client_secret: str) -> None:
     """Subscribe and print events from all the appliances."""
     client = CLIClient(client_id, client_secret)
-    async for event in client.stream_all_events():
-        rich_print(event)
+    try:
+        while True:
+            try:
+                async for event in client.stream_all_events():
+                    rich_print(event)
+            except ReadTimeout as e:
+                rich_print(f"{e} continuing...")
+            except RemoteProtocolError as e:
```

Comment:
We should think about how we want library users to handle these cases. I'd like to abstract away the httpx transport so that the library user doesn't need to be aware of it. Eg by raising library specific exceptions where needed.